source/reference/methods.txt

```diff
@@ -876,3 +876,90 @@ Role Management Methods
    * - :method:`db.updateRole()`
 
      - Updates a user-defined role.
+
+Replication Methods 
+-------------------
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Method
+
+     - Description
+ 
+   * - ``rs.printSecondaryReplicationInfo()``
+
+     - Returns the status of the secondary members of the replica set. This 
+       is identical to the ``db.printSecondaryReplicationInfo()`` method. 
+       The server :method:`rs.printSlaveReplicationInfo()` method is referred 
+       to as ``rs.printSecondaryReplicationInfo()`` in ``mongosh``.
+
+       The following is an example output from the 
+       ``rs.printSecondaryReplicationInfo()`` method issued on a replica set with two secondary members:
+
+       .. code-block:: sh 
+          :copyable: false 
+
+          source: rs2.example.net:27017
+          {
+            syncedTo: 'Tue Oct 13 2020 09:42:18 GMT-0700 (Pacific Daylight Time)',
+            replLag: '0 secs (0 hrs) behind the primary '
+          }
+          ---
+          source: rs3.example.net:27017
+          {
+            syncedTo: 'Tue Oct 13 2020 09:42:18 GMT-0700 (Pacific Daylight Time)',
+            replLag: '0 secs (0 hrs) behind the primary '
+          }
+
+   * - ``db.printSecondaryReplicationInfo()``
+
+     - Returns the status of the secondary members of the replica set. This 
+       is identical to the ``rs.printSecondaryReplicationInfo()`` method. 
+       The server :method:`db.printSlaveReplicationInfo()` method is referred 
+       to as ``db.printSecondaryReplicationInfo()`` in ``mongosh``.
+
+       The following is an example output from the 
+       ``rs.printSecondaryReplicationInfo()`` method issued on a replica set with two secondary members:
+
+       .. code-block:: sh 
+          :copyable: false 
+
+          source: rs2.example.net:27017
+          {
+            syncedTo: 'Tue Oct 13 2020  09:37:28 GMT-0700 (Pacific Daylight Time)',
+            replLag: '0 secs (0 hrs) behind the primary '
+          }
+          ---
+          source: rs3.example.net:27017
+          {
+            syncedTo: 'Tue Oct 13 2020  09:37:28 GMT-0700 (Pacific Daylight Time)',
+            replLag: '0 secs (0 hrs) behind the primary '
+          }
+
+   * - :method:`db.getReplicationInfo()`
+
+     - Returns the status of the replica set from the oplog data.
+
+   * - ``rs.isMaster()``
+
+     - Returns the replica set configuration, role, and status for 
+       the ``mongod`` instance that ``mongosh`` is connected to. The ``mongosh`` ``rs.isMaster()`` method is identical to the server :manual:`isMaster </reference/command/isMaster/>` command.
```

Comment:
I think rather than saying it is identical to the server isMaster command, it might be more accurate to say 

```
The ``mongosh`` ``rs.isMaster()`` method wraps the :dbcommand:`isMaster` command.
```

They're technically speaking not identical in behavior, although the output is the same.